bbc/simorgh
# Add metadata (min scale & link rel canonical)

As a result of the amp validate script work (https://github.com/bbc/simorgh/pull/492) I saw that we are missing some metadata for our page that we should include, even for non-AMP pages. 
This PR:
- Adds minimal scale to meta content
- Adds link rel canonical


**Testing instructions**
```
git checkout add-metadata
npm run dev
```
visit `view-source:http://localhost:7080/article/scenario-27` and see the updated metadata
e.g. 
`<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1"/>`
`<link data-react-helmet="true" rel="canonical" href="https://www.bbc.com/news/article/c0000000027o"/>`
